docs/UserGuide.md

```diff
@@ -1,4 +1,4 @@
-# AddressBook Level 4 - User Guide
+# Main - User Guide
```

Comment:
In the below, need to update the screenshot to a mock up of the UI you plan to implement. Later can be replaced by the actual screenshot.